packages/app/src/systems/Home/pages/Receive/Receive.tsx

```diff
@@ -0,0 +1,33 @@
+import { cssObj } from '@fuel-ui/css';
+import { Flex } from '@fuel-ui/react';
+import { useNavigate } from 'react-router-dom';
+
+import { ReceiverQRCode } from '../../components/QRCode';
+import { UserAddressCard } from '../../components/UserAddressCard';
+
+import { useAccount } from '~/systems/Account';
+import { Layout, Pages } from '~/systems/Core';
+
+export function Receive() {
+  const navigate = useNavigate();
+  const { account } = useAccount();
+  return (
+    <Layout title="Receive">
+      <Layout.TopBar onBack={() => navigate(Pages.wallet())} />
+      <Layout.Content>
+        <Flex css={styles.contentWrapper}>
+          <UserAddressCard account={account?.address as string} />
```

Comment:
we're using too many `as string` . we should avoid it.

can use a conditional before the render to make sure account is there:

```
{ account && (
<UserAddressCard account={account?.address} />
) }
```